tests/bluetooth/tester/src/btp_gap.c

```diff
@@ -1460,9 +1500,11 @@ static struct bt_le_per_adv_sync_cb pa_sync_cb = {
 };
 
 #if defined(CONFIG_BT_PER_ADV)
-static struct bt_data padv[10];
+static struct bt_data per_adv_local[CONFIG_BT_EXT_ADV_MAX_ADV_SET][10];
+static uint8_t per_adv_local_len[CONFIG_BT_EXT_ADV_MAX_ADV_SET];
```

Comment:
Suggest to put this into a single struct instead of having 2 arrays referring to the same thing

Reply:
Removed it, as it was only used one place and there it was hard coded to use [0]